src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -195,7 +195,10 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
 
     protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
     {
-        $tags = new FixerOptionBuilder('tags', 'The tags that should be aligned.');
+        $tags = new FixerOptionBuilder('tags', 'The tags that should be aligned. Allowed values:
+- tags with name: `[\''.implode('\', \'', self::TAGS_WITH_NAME).'\']`
+- tags with method signature: `[\''.implode('\', \'', self::TAGS_WITH_METHOD_SIGNATURE).'\']`
+- any custom tag with description e.g. `@tag <desc>`.');
```

Comment:
Hmmmm, my suggestion [does not work good](https://github.com/kayw-geek/PHP-CS-Fixer/blob/3d2cf52172baeb014242bd67f4516c4f4d8f95a0/doc/rules/phpdoc/phpdoc_align.rst)

<img width="1044" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/cdd76d08-6ab3-419d-9be6-5bcae5abb73c">

Instead of a list probably we need a regular sentence (with each group's values in parentheses?).

```suggestion
        $tags = new FixerOptionBuilder('tags', 'The tags that should be aligned. Allowed values are tags with name (`[\''.implode('\', \'', self::TAGS_WITH_NAME).'\']`), tags with method signature (`[\''.implode('\', \'', self::TAGS_WITH_METHOD_SIGNATURE).'\']`) and any custom tag with description (e.g. `@tag <desc>`).');
```

Sorry for changing conception, but I did not verify the effect locally before.

Reply:
OK, I previewed these doc files, they seem to work weel.